### DISCUSSION TITLE
Scrum Prep Milestone 3 - Week 2 (Nov 3-Nov 8)

### DISCUSSION THREAD

**Discussion by nelson-escalona** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#155] [Scrum Preps]
Scrum Prep Milestone 3 - Week 2 (Nov 3-Nov 8)

**Comment by nelson-escalona** [comment]
Worked on College Planning team backlog and created a couple of tasks for the team.

**Comment by PedroRodz** [comment]
Did not work on this class this week

**Comment by Gian-Go** [comment]
Took attendance for the team meeting during class.
[SCRUM 11_8_2024 - WEEK 2.pdf](https://github.com/user-attachments/files/17682971/SCRUM.11_8_2024.-.WEEK.2.pdf)

**Comment by christianmedina-dev** [comment]
Present on Team Meeting, organizing stuff and also worked on Lecture Topic Tasks (Issue #160) & (Issue #161).

**Comment by keisanabria** [comment]
Currently am attending an academic conference outside of Puerto Rico, hence why I am unable to take the minute of this week. However, I am planning on assigning and developing a task to complete during this sprint. Other than that, no progress has been done yet.

**Comment by keisanabria** [comment]
Changed the system of taking attendance to an automated system for everyone to get the minute automatically and take attendance easier.

**Reply by keisanabria** [reply]
I also tried creating an Issue that we changed to take attendance in meetings that were in-person, but it was declined.

**Comment by Carloscaban** [comment]
Worked on two topic lecture tasks

**Comment by carlospepin23** [comment]
Worked on assigned tasks, but not much progress done because of the Election Days Offs.

**Comment by cristianMarcial** [comment]
defined MLS tasks
